Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,33 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
-        {
-            return new Item[0];
-        }
+        public List<Item> Items { get; set; }
 
         public Inventory()
         {
-            _items = new Item[0];
+            //Items = new List<Item>(); 
```

Comment:
oke,, wierd that the tests are saying it works.

Reply:
Because you're not actually using the inventory, you're using the List of Items directly on the player. so the tests passing are slightly misleading :)

Reply:
oke

When I change the add method to this : 
```
 public void Add(Item item)
        {
            _inventory.AddItem(item);  
        }
```

then the tests is failing with this : 

```
BootCamp.Chapter.Tests.ShopTests.Add_New_Item_Adds_Item_To_Stock
   Source: ShopTests.cs line 22
   Duration: 107 ms

  Message: 
    Expected _shop.Items to contain 
    
    BootCamp.Chapter.Item
    {
       Name = "Shortsword"
       Price = 10M
       Weight = 2F
    }, but found <null>.
``` 

so it looks to me that the shop is using Items instead of the inventory. 
or am I thinking again the wrong way.

Reply:
So as i understand the original homework the structure should be

Player
-> Player has Inventory
    -> Inventory has a list of items

Adding an item to a player adds the item to the players inventory
Getting an item from a player gets the item from the players inventory

Shop
-> Shop has a list of items

Adding an item to the shop adds the item to the list of items
Getting an item from the shop gets the item from the list of items

You do not need to use the inventory for the shop... you _can_ just not required based on Chapter2/Homework2's description.

Just ensure the following:
Player.AddItem calls Inventory.AddItem
Player.GetItem calls Inventory.GetItem

shop.AddItem adds to a list on shop
shop.getItem gets an item from a list on shop

just ensure that add and get are "talking" to the same thing. Still unsure.., lets chat on discord DMs

Reply:
oke, also solved